keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py

```diff
@@ -36,10 +34,8 @@ class GroupPointsByBoundingBoxes(base_augmentation_layer_3d.BaseAugmentationLaye
         [num of frames, num of points, num of point features].
         The first 5 features are [x, y, z, class, range].
       bounding_boxes: 3D (multi frames) float32 Tensor with shape
-        [num of frames, num of boxes, num of box features]. Boxes are expected
-        to follow the CENTER_XYZ_DXDYDZ_PHI format. Refer to
-        https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/formats.py
-        for more details on supported bounding box formats.
+        [num of frames, num of boxes, num of box features].
+        The first 8 features are [x, y, z, dx, dy, dz, phi, box class].
```

Comment:
quick question; so as we move to the dictionary format for our standard boxes should we implement 3d boxes in dictionary format too?

I think this would mean moving class to a different key

Reply:
Yes, we should move this to dictionary format. Right now, all of the code is using non-dict format, so the box format in this PR is just made to match that.